clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/CatsSearchProvider.groovy

```diff
@@ -112,7 +112,12 @@ class CatsSearchProvider implements SearchProvider {
     List<Map<String, String>> results = resultPage.findResults { String key ->
       Map<String, String> result = providers.findResult { it.parseKey(key) }
       if (result) {
-        return searchResultHydrators.containsKey(result.type) ? searchResultHydrators[result.type].hydrateResult(cacheView, result, key) : result
+        def resultResource = new SearchableResource(resourceType: result.type, platform: result.provider)
```

Comment:
You'll want to ensure both `resourceType` and `platform` values are the same case as the ones from the providers, otherwise your `hashCode()` values will be off, and they won't be equal